src/components/ListItem.js

```diff
@@ -1,37 +1,52 @@
 import React, { useState } from 'react';
 import { ITEMS, USERS } from '../constants';
 import { db } from '../lib/firebase';
-import { getDifferenceInHours } from '../lib/getDifferenceInHours';
+import { getDifferenceInHours, getDifferenceInDays } from '../lib/timeUtils';
+import calculateEstimate from '../lib/estimates';
+
 import '../CSS/ListItem.css';
 
 const ListItem = props => {
-  let hoursDiff = getDifferenceInHours(props.item.last_purchased);
+  const { item, token } = props;
```

Comment:
Nice!  You could also de-structure the props in line 9, if you wanted. `const ListItem = ({ item, token }} => {`, but either works just fine.